doc/source/data/shuffling-data.rst

```diff
@@ -128,13 +128,7 @@ to data transfer costs. This cost can be prohibitive when using very large datas
 
 The best route for determining the best tradeoff between preprocessing time and cost and
 per-epoch shuffle quality is to measure the precision gain per training step for your
-particular model under different shuffling policies:
-
-* no shuffling,
-* local (per-shard) limited-memory shuffle buffer,
-* local (per-shard) shuffling,
-* windowed (pseudo-global) shuffling, and
-* fully global shuffling.
+particular model under different shuffling policies such as no shuffling, local shuffling, or global shuffling.
```

Comment:
It seems that there are 3 shuffling policies, how do different shuffling options get mapped to them?
- Shuffle the ordering of files > ?
- Local shuffle when iterating over batches > local shuffling?
- Shuffling block order > ? 
- Shuffling all rows > global shuffling


If we want to categorize them like this, why organize the content in this order?
- no shuffling
- local shuffling
  - ...
  - ...
- global shuffling

Reply:
Frankly, this section is much more academic/algorithmically impactful.

On the other hand, the section above is more about the different shuffle policies that you can use Ray Data for, which have different *system requirements*. 

We don't need to make them correspond IMO.

Reply:
okay. I'm just sharing the questions in my mind after looking at this. Maybe they don't really matter
Will leave this up to u